dbinit.py

```diff
@@ -47,6 +47,7 @@ def dbinit():
     lab1.set_lab_guide_md(
             '# Lab guide for Hello World\r\n\r\nThis is a lab guide for Hello World\r\n\r\n## 1. Make sure this lab is working\r\n\r\nTo make sure this lab is working, you should open the helloworld service as ilustrated below:\r\n\r\n![open-service](https://raw.githubusercontent.com/hackinsdn/labs/refs/heads/main/lab00-helloworld/images/helloworld-open-service.png)\r\n\r\nYou should see something like this:\r\n\r\n![service](https://raw.githubusercontent.com/hackinsdn/labs/refs/heads/main/lab00-helloworld/images/helloworld-service.png)\r\n\r\n> [!IMPORTANT]  \r\n> The hello world service shows a message indicating the status of the system. Which option below best describe the status indicated by the hello world service?\r\n>\r\n> <input type="radio" name="answer_helloworld_q1" id="id1" value="system-degradated" /> <label for="id1">System is working in a degradated status.</label><br>\r\n> <input type="radio" name="answer_helloworld_q1" id="id2" value="system-not-working" /> <label for="id2">System is not working.</label><br>\r\n> <input type="radio" name="answer_helloworld_q1" id="id3" value="system-working" /> <label for="id3">System is working correctly.</label><br>\r\n> <input type="radio" name="answer_helloworld_q1" id="id4" value="system-unknow" /> <label for="id4">It is not possible to know the system status.</label><br>\r\n\r\n## 2. Accessing the lab console\r\n\r\nSometimes you will need to run commands on some components of the Lab. Follow the indication below to run commands on the Lab container:\r\n\r\n![open-terminal](https://raw.githubusercontent.com/hackinsdn/labs/refs/heads/main/lab00-helloworld/images/helloworld-open-term.png)\r\n\r\nYou should see something like this:\r\n\r\n![terminal](https://raw.githubusercontent.com/hackinsdn/labs/refs/heads/main/lab00-helloworld/images/helloworld-terminal.png)\r\n\r\n> [!IMPORTANT]  \r\n> When you clicked on the link indicated above, which component was loaded from the Lab:\r\n>\r\n> <select name="answer_helloworld_q2">\r\n>  <option value="">--</option>\r\n>  <option>The Kubernetes cluster</option>\r\n>  <option>The terminal of the container</option>\r\n>  <option>Nothing was opened</option>\r\n>  <option>All options are correct</option>\r\n> </select>'
     )
+    lab1.allowed_groups.append(everybody)
```

Comment:
took the opportunity to also fix this, since now labs are only showed for authorized groups, the HelloWorld Lab was added so that new users will have access right away